src/emulator/util/CMakeLists.txt

```diff
@@ -1,6 +1,12 @@
 add_library(
 	util
-	INTERFACE
+	STATIC
```

Comment:
I think the initialisation code to set up the logger actually belongs in `emulator` because nothing else needs to call it, and I think we should be passing the logger pointer around to wherever needs it rather than having it as a global.

Reply:
Honestly I've tried doing that before and it's... not great. I think if we should have one global, logging has to be it. It just makes everything much easier and more readable.
I think using a global is justified in that case and every project I've come across does it this way.